### ISSUE TITLE
Minor/cosmetic issue re consistency of naming CIs

### ISSUE THREAD

**Issue by andymilne** [paul-buerkner/brms#826]
The model summary uses `l-95% CI` and `u-95% CI`; bayes_R2 uses `Q2.5` and `Q97.5`; loo_R2 doesn't show the CIs at all. I think I have seen something like `upper CI` and `lower CI` used too, but I cannot find where right now. Also, it might be worth reporting the R2 values to 2 decimal places, as are all the values in the main model summary.

**Comment by paul-buerkner** [comment]
This is true. Eventually, we will use the posterior package (https://github.com/jgabry/posterior/issues) as the backend for all such summeries so that most of them will be aligned in brms 3. I am not aware that loo_R2 emits any CIs.
